### PR TITLE
feat: [BiW Editor] Replace keyboard shortcut behaviour + Re-design PublicationDetailsView

### LINKED ISSUE #521: BiW replace keyboard shortcut behaviour
While standing on any land in the current implementation of BiW, the K shortcut just enters edit mode and start editing the Land even if its not yours. While this is the preferred situation while development, we need to be able to test other scenarios.

- [x] Disable this "god mode editing" while pressing K
- [x] Pressing K in a Land without owner or contribution permissions should display a message saying "This land does not belong to you, nor have you been granted operating permits by its owner."
- [x] Pressing K in a Land where you have permission to edit (either by ownership or operation) will trigger the editor.
- [x] Pressing K in a permitted Scene, that was created with other system (SDK) will display the next message "This place was created with the SDK and can not be edited in-world".

### PR DESCRIPTION
**Pull request by sandrade-dcl**
### Fixes #521
While standing on any land in the current implementation of BiW, the K shortcut just enters edit mode and start editing the Land even if its not yours. While this is the preferred situation while development, we need to be able to test other scenarios.

- [x] Disable this "god mode editing" while pressing K
- [x] Pressing K in a Land without owner or contribution permissions should display a message saying "This land does not belong to you, nor have you been granted operating permits by its owner."

image

- [x] Pressing K in a Land where you have permission to edit (either by ownership or operation) will trigger the editor.
- [x] Pressing K in a permitted Scene, that was created with other system (SDK) will display the next message "This place was created with the SDK and can not be edited in-world"

image

### Fixes #537 
- [x] Add new design for PublicationDetailsView

image

## How to test it?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:feat/BiW-Replace-keyboard-shortcut-behaviour&ENV=zone&ENABLE_BUILDER_IN_WORLD&position=80,154
2. Press 'K' to open the Builer In World editor.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by pravusjif — APPROVED** [review]

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs [hunk 1]
```diff
@@ -55,13 +58,19 @@ public class BuilderInWorldController : MonoBehaviour
     [HideInInspector]
     public bool isBuilderInWorldActivated = false;
 
+    private const float RAYCAST_MAX_DISTANCE = 10000f;
+    private const string LAND_EDITION_NOT_ALLOWED_BY_PERMISSIONS_MESSAGE = "This land does not belong to you, nor have you been granted operating permits by its owner.";
+    private const string LAND_EDITION_NOT_ALLOWED_BY_SDK_LIMITATION_MESSAGE = "This place was created with the SDK and can not be edited in-world.";
+    private const float CACHE_TIME_LAND = 5 * 60;
+    private const float CACHE_TIME_SCENES = 1 * 60;
+    private const float REFRESH_LANDS_WITH_ACCESS_INTERVAL = 2 * 60;
```
Comment (Suduck) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs:66]: Should this be moved to the settings? At least the cache, this way we will have all the settings in the same place

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs [hunk 2]
```diff
@@ -615,4 +653,39 @@ private void OnPlayerTeleportedToEditScene(Vector2Int coords)
...
+        updateLandsWithAcessCoroutine = CoroutineStarter.Start(CheckLandsAccess());
+    }
+
+    private IEnumerator CheckLandsAccess()
+    {
+        while (true)
+        {
+            UpdateLandsWithAccess();
+            yield return WaitForSecondsCache.Get(REFRESH_LANDS_WITH_ACCESS_INTERVAL);
+        }
```
Comment (Suduck) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs:669]: I am worried about to overload the servers with this, what do you think to deactivate this if you are already editing a scene/land?

**Review by Suduck — CHANGES_REQUESTED** [review]
I am worried about these messages stacking, should we add an automatic remove after X time at least?

image

Also, there is a bug that maybe is not directly related to this but this change has made it more clear

mvp4

**Review by Suduck — APPROVED** [review]

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs (hunk 1 above)
Comment (sandrade-dcl) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs]: Done in https://github.com/decentraland/unity-renderer/pull/563/commits/d8f45c487609117aed1f93f4a885e3c9fba323ce. Thanks!

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs (hunk 2 above)
Comment (sandrade-dcl) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs]: Done in https://github.com/decentraland/unity-renderer/pull/563/commits/c600fe6b2326349349eb82200cc003b3aa31acf6. Thanks!

**Pull request merged**